packages/commonwealth/client/scripts/views/components/Header/LoginSelector.tsx

```diff
@@ -51,6 +52,12 @@ export const LoginSelector = () => {
   const [isTOSModalOpen, setIsTOSModalOpen] = useState(false);
   const [isJoined, setIsJoined] = useState(false);
 
+  useEffect(() => {
+    NewProfilesController.Instance.isFetched.on('redraw', () => {
```

Comment:
So, this event works but the other one doesn't? Why?

Reply:
Looking into this now, will do a writeup when I get to the bottom of it. I think it has something to do with the react useEffect initialization times. It is strange because adding breakpoints to this, the useAuthor, and the emitting from the newProfilesController shows this one being hit, but the author one not being hit.

Anyways will do a writeup with the findings to finish off this ticket.

Reply:
Great. I think it's important we understand the issue with that one case, so that we don't miss deeper bugs down the road. I'll mark this comment resolved for now, please post the writeup as a comment on the PR directly.